src/index.js

```diff
@@ -8,13 +8,7 @@ const mergeFolders = require('./merge-folders').merge
 function buildBundle (config) {
   return rollup.rollup(config)
     .then(function (bundle) {
-      return bundle.write({
-        format: config.format || 'es6',
```

Comment:
"If you do require to have "format: es6" then just create a new object, put the property in there and then just copy all remaining properties from initial config." - @YuryStrozhevsky my mentor @tmornini would be very proud to hear you say that. He's always telling me "Objects are cheap. Just make another one with the permutation and call it a day."